lib/keyring_liberator.rb

```diff
@@ -63,7 +63,7 @@ def self.get_all_keyrings
     end
 
     def self.get_keyring_at_path(path)
-      Keyring.from_hash(YAML.load(path.read)) if path.file?
+      Keyring.from_hash(YAML.safe_load(path.read)) if path.file?
```

Comment:
any chance you can check for `YAML.safe_load`'s existence before using it? in the CI it doesn't seem to exist (could be a ruby 2.0 thing )